### ISSUE TITLE
Missing appdata and screenshots for AppStream

### ISSUE THREAD

**Issue by dwrobel** [borco/plantumlqeditor#8]
Hi,

I'd like to package the plantumlqeditor software for Fedora. The packaging is completed now in ~95% and available in the [copr](https://copr.fedorainfracloud.org/coprs/dwrobel/plantumlqeditor/) repository. The last missing point is a lack of [appdata](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps) file, and more precisely screenshots with required resolution. According to [4.1.3. Recommended metadata file contents](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps): 

> All screenshots should have a 16:9 aspect ratio, and should have a width that is no smaller than 620px
> Screenshots should be in PNG or JPEG format. PNG is the preferred format;

I would appreciate if you can create one to three screenshots of plantumlqeditor as that would allow the software to appear in the Software Centers (e.g. [GNOME](https://wiki.gnome.org/Apps/Software)).

A ready to use example of appdata file can be taken [here](https://dwrobel.fedorapeople.org/projects/rpmbuild/SOURCES/plantumlqeditor.appdata.xml) the thing which needs to be adjusted are links to new screenshots which now points to SF page which have images with too small width and height:
```
$ appstream-util validate-relax plantumlqeditor.appdata.xml 
plantumlqeditor.appdata.xml: FAILED:
• attribute-invalid     : <screenshot> width too small [https://a.fsdn.com/con/app/proj/plantumlqeditor/screenshots/mainwindow.png/182/137/1]
• attribute-invalid     : <screenshot> height too small [https://a.fsdn.com/con/app/proj/plantumlqeditor/screenshots/mainwindow.png/182/137/1]
• attribute-invalid     : <screenshot> width too small [https://a.fsdn.com/con/app/proj/plantumlqeditor/screenshots/mainwindow--sequence-assistant.png/182/137/1]
• attribute-invalid     : <screenshot> height too small [https://a.fsdn.com/con/app/proj/plantumlqeditor/screenshots/mainwindow--sequence-assistant.png/182/137/1]
• attribute-invalid     : <screenshot> width too small [https://a.fsdn.com/con/app/proj/plantumlqeditor/screenshots/propertiesdialog.png/182/137/1]
• attribute-invalid     : <screenshot> height too small [https://a.fsdn.com/con/app/proj/plantumlqeditor/screenshots/propertiesdialog.png/182/137/1]
Validation of files failed
```

Once the correct images will be used then the validation process should pass without failures.

**Comment by ghost** [comment]
Hi,

The original images are stored in [docs](https://github.com/borco/plantumlqeditor/tree/master/docs):

https://github.com/borco/plantumlqeditor/blob/master/docs/mainwindow--sequence-assistant.png
https://github.com/borco/plantumlqeditor/blob/master/docs/mainwindow.png
https://github.com/borco/plantumlqeditor/blob/master/docs/propertiesdialog.png

From what I see, they should be large enough. Please let me know if you need something else.

Cheers.

**Comment by ghost** [comment]
I've updated those images with newer versions.

**Comment by dwrobel** [comment]
>Please let me know if you need something else.

Thank you for a prompt reply, this solved the issue, please find PR #9 with updated appdata.

**Comment by ghost** [comment]
appdata merged
